app/models/patient_import_row.rb

```diff
@@ -405,6 +405,8 @@ def validate_first_name
         first_name.header,
         "is greater than #{MAX_FIELD_LENGTH} characters long"
       )
+    elsif !first_name.to_s.match?(PDS::Patient::VALID_NAME_REGEX)
+      errors.add(first_name.header, "includes invalid character(s).")
```

Comment:
Minor content thing, I think we've normally not put a full stop at the end of these errors, but maybe we should add it consistently across all of them?

Reply:
Let's make it consistent for now. Can ask the designers after